test/test_ops.py

```diff
@@ -285,6 +284,7 @@ def __new__(cls, name, variant_test_name=""):
             # AllowedOpInfoEntry('exp'),
             # AllowedOpInfoEntry('diag_embed'),
             # AllowedOpInfoEntry('diagonal'),
+            # AllowedOpInfoEntry('einsum'),
```

Comment:
nit: can you add the link to the github issue in the comment?

Reply:
Done